go/vt/wrangler/split.go

```diff
@@ -118,14 +107,24 @@ func (wr *Wrangler) WaitForFilteredReplication(ctx context.Context, keyspace, sh
 		}
 
 		delaySecs := stats.SecondsBehindMasterFilteredReplication
-		lastSeenDelay := time.Duration(delaySecs) * time.Second
+		lastSeenDelay = time.Duration(delaySecs) * time.Second
 		if lastSeenDelay < 0 {
 			return fmt.Errorf("last seen delay should never be negative. tablet: %v delay: %v", alias, lastSeenDelay)
 		}
 		if lastSeenDelay <= maxDelay {
 			wr.Logger().Printf("Filtered replication on tablet: %v has caught up. Last seen delay: %.1f seconds\n", alias, lastSeenDelay.Seconds())
-			return nil
+			return io.EOF
 		}
 		wr.Logger().Printf("Waiting for filtered replication to catch up on tablet: %v Last seen delay: %.1f seconds\n", alias, lastSeenDelay.Seconds())
+		return nil
+	})
+	if err != nil {
+		return fmt.Errorf("could not stream health records from tablet: %v err: %v", alias, err)
+	}
+	select {
```

Comment:
Please add a newline here to better separate this code from the StreamHealth call + error check.